src/src/LeftMenu/MenuListItem/MenuListItem.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import { mount } from "enzyme";
+import React from "react";
+import MenuListItem from "./MenuListItem";
+
+describe("MenuListItem", () => {
+  it("should render list item with text", () => {
+    const component = mount(<MenuListItem className="test">Test</MenuListItem>);
+    expect(component.text()).toEqual("Test");
```

Comment:
What are you testing here? It's looks like you're testing React's functionality...